packages/core/src/components/tooltip/tooltip.stories.ts

```diff
@@ -0,0 +1,65 @@
+import { storiesOf } from '@storybook/web-components';
+import { html } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined';
+import { v4 as uuidv4 } from 'uuid';
+
+// @ts-ignore
+import readme from './readme.md';
+
+const positions = ['top', 'right', 'bottom', 'left'] as const;
+
+interface TooltipArgs {
+  position: typeof positions;
+  label: string;
+  id?: string;
+}
+
+const Tooltip = ({ position, label, id }: TooltipArgs) => html`
+  <dso-tooltip position=${position} for=${ifDefined(id)}>
+    ${label}
+  </dso-tooltip>
+`;
+
+const stories = storiesOf('Tooltip', module)
+  .addParameters({
+    docs: {
+      page: readme
+    },
+    argTypes: {
+      position: {
+        options: positions,
+        control: {
+          type: 'select'
+        }
+      }
+    },
+    args: {
+      position: 'bottom',
+      label: "I'm a tooltip"
+    }
+  });
+
+stories.add(
+  'as child',
+  ({ position, label }: any) => html`
+    <button type="button">
+      <span>Hover or focus me</span>
```

Comment:
Ik zie in storybook een wijziging van de positioning niet live doorkomen

Reply:
Dat klopt, dat is stuk in het web component

Reply:
Zie #1090